packages/uniorg-parse/src/parser.ts

```diff
@@ -999,9 +1044,12 @@ class Parser {
   }
 
   private parseClock(): Clock {
-    this.r.advance(this.r.forceMatch(/^[ \t]*CLOCK:[ \t]*/));
+    const begin = this.r.offset();
+    const parsedClock = this.r.forceMatch(/^[ \t]*CLOCK:[ \t]*/);
+    this.r.advance(parsedClock);
     const value = this.parseTimestamp();
 
+    const end = begin + parsedClock.input.length;
```

Comment:
❗🐛 this does not take into account advancement from `this.parseTimestamp`. `parsedClock` is only initial `CLOCK: ` string
```suggestion
    const begin = this.r.offset();
    this.r.advance(this.r.forceMatch(/^[ \t]*CLOCK:[ \t]*/));
    const value = this.parseTimestamp();
    const end = this.r.offset();
```